dae/dae/studies/study_sections_config.py

```diff
@@ -0,0 +1,34 @@
+from dae.studies.people_group_config_parser import PeopleGroupConfigParser
+from dae.studies.genotype_browser_config_parser import \
+    GenotypeBrowserConfigParser
+
+
+class StudySectionsConfigMixin(object):
```

Comment:
probably should be moved into StudyConfigParserBase

Reply:
I moved all of his methods to study config.